.github/label-actions.yml

```diff
@@ -4,6 +4,7 @@ devnet-e2e-test:
     comment: The CI will now also run the e2e tests on devnet, which increases the time it takes to complete all CI checks.
     label:
       - devnet
+      - push-image
```

Comment:
I thought this would be inherited from the actions of the `devnet` label, it was not, unfortunately.